draft-ietf-core-senml.md

```diff
@@ -718,63 +726,77 @@ number of this specification.
+| mol      | mole                                              | float | RFC-AAAA  |
+| Hz       | hertz                                             | float | RFC-AAAA  |
+| rad      | radian                                            | float | RFC-AAAA  |
+| sr       | steradian                                         | float | RFC-AAAA  |
+| N        | newton                                            | float | RFC-AAAA  |
+| Pa       | pascal                                            | float | RFC-AAAA  |
+| J        | joule                                             | float | RFC-AAAA  |
+| W        | watt                                              | float | RFC-AAAA  |
+| C        | coulomb                                           | float | RFC-AAAA  |
+| V        | volt                                              | float | RFC-AAAA  |
+| F        | farad                                             | float | RFC-AAAA  |
+| Ohm      | ohm                                               | float | RFC-AAAA  |
+| S        | siemens                                           | float | RFC-AAAA  |
+| Wb       | weber                                             | float | RFC-AAAA  |
+| T        | tesla                                             | float | RFC-AAAA  |
+| H        | henry                                             | float | RFC-AAAA  |
+| Cel      | degrees Celsius                                   | float | RFC-AAAA  |
+| lm       | lumen                                             | float | RFC-AAAA  |
+| lx       | lux                                               | float | RFC-AAAA  |
+| Bq       | becquerel                                         | float | RFC-AAAA  |
+| Gy       | gray                                              | float | RFC-AAAA  |
+| Sv       | sievert                                           | float | RFC-AAAA  |
+| kat      | katal                                             | float | RFC-AAAA  |
+| m2       | square meter (area)                               | float | RFC-AAAA  |
+| m3       | cubic meter (volume)                              | float | RFC-AAAA  |
+| l        | liter (volume)*                                   | float | RFC-AAAA  |
+| m/s      | meter per second (velocity)                       | float | RFC-AAAA  |
+| m/s2     | meter per square second (acceleration)            | float | RFC-AAAA  |
+| m3/s     | cubic meter per second (flow rate)                | float | RFC-AAAA  |
+| l/s      | liter per second (flow rate)*                     | float | RFC-AAAA  |
+| W/m2     | watt per square meter (irradiance)                | float | RFC-AAAA  |
+| cd/m2    | candela per square meter (luminance)              | float | RFC-AAAA  |
+| bit      | bit (information content)                         | float | RFC-AAAA  |
+| B        | Byte (information content, note 3)                | float | RFC-AAAA  |
```

Comment:
So I don't think we should add B because it is simply a scalar multiple of bit so same issue as having prefixes.